analysers/analyser_merge_charging_station_FR.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                                                                                    ##
+## Copyrights Noémie Lehuby 2020                                                                      ##
+##                                                                                                                                    ##
+## This program is free software: you can redistribute it and/or modify     ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or          ##
+## (at your option) any later version.                                                                    ##
+##                                                                                                                                    ##
+## This program is distributed in the hope that it will be useful,                  ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See     ##
+## the GNU General Public License for more details.                                        ##
+##                                                                                                                                    ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                                                                                    ##
+###########################################################################
+
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+
+class Analyser_Merge_Charging_station_FR(Analyser_Merge):
+    def __init__(self, config, logger=None):
+        Analyser_Merge.__init__(self, config, logger)
+        self.missing_official = self.def_class(item=9990, id=1, level=3, tags=['merge'],
+                                               title=T_('Car charging station not integrated'))
+        self.possible_merge = self.def_class(item=9991, id=3, level=3, tags=['merge'],
+                                             title=T_('Car charging station integration suggestion'))
+        self.update_official = self.def_class(item=9992, id=4, level=3, tags=['merge'],
+                                              title=T_('Car charging station update'))
+
```

Comment:
Please use the new code layout and user documentation.

See for example: https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_merge_heritage_FR_merimee.py

Reply:
See the result of the documentation items on http://beta.osmose.openstreetmap.fr

Reply:
I fixed the code layout.